tests/Util/yaml_fixtures/array_double_quoted_80_value_single.test

```diff
@@ -0,0 +1,14 @@
+caddy:
+    ports:
+      - "foo:foo"
+    volumes:
+      - caddy_data
+===
+$data['caddy']['ports'][] = "bar:baz";
+===
+caddy:
+    ports:
+      - "foo:foo"
+      - 'bar:baz'
```

Comment:
this test passes and it shouldnt. However, If I replace `"foo:foo"` w/ `"80:80"` in on both sides, the test fails because it cant find `"80:80"`